src/includes/getting-started-install/apple.watchos.mdx

```diff
@@ -11,4 +11,6 @@ target 'YourApp' do
 end
 ```
 
-Afterwards run `pod install`.
+Then run `pod install`.
+
+We also support alternate [installation methods](install/).
```

Comment:
Whatever we agree about the comment on the apple.ios.mdx would go here too

Reply:
see previous